### PR TITLE
Rails 4.2 update

### PR DESCRIPTION
**Pull request by mkasztelnik**

### CODE REVIEW

**Comment by mkasztelnik** [comment]
@VPH-Share/cyfronet-dev-team please take a look, it is deployed into dev environment

File: app/serializers/basic_endpoint_serializer.rb [hunk 1]
```diff
@@ -2,9 +2,12 @@
...
 #
 class BasicEndpointSerializer < ActiveModel::Serializer
+  include Vphshare::Application.routes.mounted_helpers
+
   attributes :id, :name, :description, :endpoint_type, :url
 
   def url
-    atmosphere.descriptor_api_v1_endpoint_url(object.id)
+    atmosphere.
+      descriptor_api_v1_endpoint_url(object.id)
```
Comment (nowakowski) [inline app/serializers/basic_endpoint_serializer.rb]: Why the line split?

Comment (mkasztelnik) [inline app/serializers/basic_endpoint_serializer.rb]: Right, I will revert it. It was done during active model serializer 0.8.3 tests.

Comment (mkasztelnik) [inline app/serializers/basic_endpoint_serializer.rb]: Fixed

**Comment by nowakowski** [comment]
:+1:

**Comment by mkasztelnik** [comment]
I needed to switch into rails `4-2-stable` branch, since in `4.2.1` there is a [bug](https://github.com/rails/rails/issues/18698) connected with not responding to `HEAD` request. Currently commit which introduces this bug is rolled back from this branch and [appropriate fix](https://github.com/rails/rails/pull/19431) is waiting to be merged. Most probably this will be fixed in rails `4.2.2`, but since it will be released we can use stable `4.2` branch.

**Pull request merged**